src/thread_timer.h

```diff
@@ -9,13 +9,14 @@ namespace internal {
 
 class ThreadTimer {
  public:
-  ThreadTimer() = default;
+  ThreadTimer(bool measure_process_cpu_time_)
```

Comment:
make this explicit please (to avoid an accidental implicit conversion).

https://google.github.io/styleguide/cppguide.html#Implicit_Conversions